releases/2.9.0/artifacts.json

```diff
@@ -1,26 +1,26 @@
 {
   "2.9.0": {
     "holoscan": {
-      "debian-version": "2.9.0.1-1",
+      "debian-version": "2.9.0.0-1",
```

Comment:
Are we expecting `2.9.0.0-1` or `2.9.0.2-1`?

Reply:
I am waiting for GA to update this file again. However, the flagged line may be out of date. It's currently set to 
```
      "debian-version": "2.9.0.2-1",
      "wheel-version": "2.9.0",
```

Reply:
\Updated with 2.9.0 GA! Thanks!